components/add_users_to_team/add_users_to_team.jsx

```diff
@@ -143,8 +143,6 @@ export default class AddUsersToTeam extends React.Component {
 
         for (let i = 0; i < users.length; i++) {
             const user = Object.assign({}, users[i]);
```

Comment:
I resisted the urge to refactor away this for loop in favour of keeping the change tight for release.